nselib/msrpctypes.lua

```diff
@@ -4523,6 +4523,26 @@ function unmarshall_SERVICE_STATUS(data, pos)
 end
 
 
+--- Unmarshalls a null-terminated Unicode string based upon a 32-bit offset (LPTSTR)
```

Comment:
Is there any significance in the offset being a 32-bit value?

Reply:
Yes, its suggested on MSDN website to prefer 32-bit buffer size.